nodebuilder/p2p/network.go

```diff
@@ -68,16 +69,31 @@ var networkAliases = map[string]Network{
 	"private": Private,
 }
 
-// listProvidedNetworks provides a string listing all known long-standing networks for things like
-// command hints.
-func listProvidedNetworks() string {
-	var networks string
-	for net := range networksList {
-		// "private" network isn't really a choosable option, so skip
+// orderedNetworks is a list of all known networks in order of priority.
+var orderedNetworks = []Network{Mainnet, Mocha, Arabica, Private}
+
+// GetOrderedNetworks provides a list of all known networks in order of priority.
+func GetOrderedNetworks() []Network {
```

Comment:
last nit: let's remove the word ordered from the method. It's redundant as slice already implies an order.  You can keep the priority ordering in the comment though

Reply:
Fair

Reply:
doesn't matter. I mostly care about public methods